.agent/docs/schema.md

```
@@ -955,28 +957,32 @@ create index idx_taxonomy_drift_pack
 - `PUBLISHED`
 - `ARCHIVED`
 
-### 11.2 `review.review_session.status`
+### 11.2 `pack.slot_definition.status` / `pack.policy_definition.status`
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)